packages/muelu/src/Graph/MatrixTransformation/MueLu_AmalgamationFactory_kokkos_decl.hpp

```diff
@@ -56,7 +56,7 @@
 #include "MueLu_Exceptions.hpp"
 
 #ifdef HAVE_MUELU_KOKKOS_REFACTOR
-#include "MueLu_AmalgamationInfo_kokkos_fwd.hpp"
+#include "MueLu_AmalgamationInfo_kokkos.hpp"
```

Comment:
Doesn't the file `MueLu_AmalgamationInfo_kokkos.hpp` include this file, `MueLu_AmalgamationFactory_kokkos_decl.hpp`?

Reply:
No, `MueLu_AmalgamationFactory_kokkos_{decl,def}.hpp` may include `MueLu_AmalgamationInfo_kokkos.hpp` but not the otherway around. Actually I think that `MueLu_AmalgamationInfo_kokkos_{decl,def}.hpp` have no reference to the `AmalgamationFactory_kokkos`.

Reply:
My mistake.  I misread `MueLu_AmalgamationInfo_kokkos` as `MueLu_AmalgamationFactory_kokkos`.